views/index.ejs

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html>
 <head>
+  <!--localhost:3000 to test-->
```

Comment:
Could you remove this line?

Reply:
Removed line, moved pull request to dev branch